scripts/Dockerfile

```diff
@@ -88,3 +88,20 @@ COPY --from=builder --chown=runner:runners /build/${APP_HOME}/public /webapp/pub
 COPY --from=builder --chown=runner:runners /build/${APP_DIST_HOME} /webapp/.next
 
 ENTRYPOINT [ "npx", "next", "start", "-p", "4200" ]
+
+FROM nginx:1.19.2-alpine as output-static
+
+ARG GIT_BRANCH
+ARG GIT_SHA
+LABEL branch=${GIT_BRANCH}
+LABEL commit=${GIT_SHA}
+ENV GIT_BRANCH=${GIT_BRANCH}
+ENV GIT_SHA=${GIT_SHA}
+ARG APP
```

Comment:
minor but i like all `ARG`s at the top so you immediately see what build args you need

Reply:
fixed